lusSTR/tests/test_suite.py

```diff
@@ -136,83 +147,143 @@ def test_indel_flag(locus, sequence, uas, kit, output):
-    )
-    assert os.file.exists(
-        f"{tmp_path}/Separated_lusstr_Files/UAS_bulk_test/Positive_Control2{sex}.txt"
-    )
+def test_annotate_sr_sexloci(input, testoutput, flank_output, kit, tmp_path):
+    str_path = str(tmp_path / "WD")
+    inputfile = data_file(f"{input}.csv")
+    inputfile_sex = data_file(f"{input}_sexloci.csv")
+    exp_sex_out = data_file(testoutput)
+    exp_sex_flank_out = data_file(flank_output)
+    obs_sex_out = str(tmp_path / "WD/testformatsr_sexloci.txt")
+    obs_sex_flank_out = str(tmp_path / "WD/testformatsr_sexloci_flanks_anno.txt")
+    if kit == "forenseq":
+        arglist = [
+            "config",
+            "-w",
+            str_path,
+            "-o",
+            "testformatsr",
+            "--sex",
+            "--input",
+            "WD",
+            "--straitrazor",
+        ]
+    else:
+        arglist = [
+            "config",
+            "-w",
+            str_path,
+            "-o",
+            "testformatsr",
+            "--sex",
+            "--input",
+            "WD",
+            "--straitrazor",
+            "--powerseq",
+        ]
+    lusSTR.cli.main(lusSTR.cli.get_parser().parse_args(arglist))
+    shutil.copyfile(inputfile_sex, os.path.join(str_path, "testformatsr_sexloci.csv"))
+    shutil.copyfile(inputfile, os.path.join(str_path, "testformatsr.csv"))
+    annot_arglist = ["strs", "annotate", "-w", str_path]
+    lusSTR.cli.main(lusSTR.cli.get_parser().parse_args(annot_arglist))
+    assert filecmp.cmp(exp_sex_out, obs_sex_out) is True
+    assert filecmp.cmp(exp_sex_flank_out, obs_sex_flank_out) is True
+
+
+def test_config(tmp_path):
+    obs_config = str(tmp_path / "config.yaml")
+    exp_config = resource_filename("lusSTR", "data/config.yaml")
+    arglist = ["config", "-w", str(tmp_path)]
+    lusSTR.cli.main(lusSTR.cli.get_parser().parse_args(arglist))
+    assert os.path.exists(obs_config)
+
+
+def test_config_settings(tmp_path):
+    obs_config = str(tmp_path / "config.yaml")
+    arglist = ["config", "-w", str(tmp_path), "--straitrazor", "--reference", "--ce"]
+    lusSTR.cli.main(lusSTR.cli.get_parser().parse_args(arglist))
+    with open(obs_config, "r") as file:
+        data = yaml.safe_load(file)
+    assert data["uas"] is False
+    assert data["data_type"] == "ce"
+    assert data["profile_type"] == "reference"
+
+
+@pytest.mark.parametrize(
+    "command, output, format_out, annot_out, all_out",
+    [
+        ("format", "lusstr_output.csv", True, False, False),
+        ("annotate", "lusstr_output.txt", True, True, False),
+        ("all", "lusstr_output/Positive_Control_evidence_ngs.csv", True, True, True),
+    ],
+)
+def test_snakemake(command, output, format_out, annot_out, all_out, tmp_path):
+    config = str(tmp_path / "config.yaml")
+    inputfile = data_file("UAS_bulk_input/Positive Control Sample Details Report 2315.xlsx")
+    exp_output = data_file(output)
```

Comment:
Running this data through `lusstr strs all` on the command line went seamlessly for me.